src/main/java/com/epam/controller/LoginController.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.controller;
+
+import com.epam.dao.UserDao;
+import com.epam.model.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+public class LoginController {
+    private final UserDao userDao;
+
+    @GetMapping("/login")
+    public ResponseEntity<String> login(@RequestParam String username, @RequestParam String password) {
+        Optional<User> optionalUser = userDao.findByUsername(username);
+        if (optionalUser.isPresent()) {
+            User user = optionalUser.get();
+            if (user.getPassword().equals(password)) {
+                return ResponseEntity.ok("Login successful. Status: 200 OK");
+            }
+        }
+        return ResponseEntity
+                .status(HttpStatus.UNAUTHORIZED)
+                .body("Invalid credentials. Status: 401 Unauthorized");
+    }
+
+    @PutMapping("/change-password")
+    public ResponseEntity<String> changePassword(
+            @RequestParam String username,
+            @RequestParam String oldPassword,
+            @RequestParam String newPassword) {
+        Optional<User> optionalUser = userDao.findByUsername(username);
+        if (optionalUser.isPresent()) {
+            User user = optionalUser.get();
+            if (user.getPassword().equals(oldPassword)) {
+                user.setPassword(newPassword);
```

Comment:
Maybe you'll need to do some checks, before setting a new password? I mean check if the password matches some criteria? I'm not sure if it's a part of the task, but just to take your attention.